compiler/arm/codegen/FPTreeEvaluator.cpp

```diff
@@ -3138,4 +3138,14 @@ TR::Register *OMR::ARM::TreeEvaluator::fRegStoreEvaluator(TR::Node *node, TR::Co
    return NULL;
    }
 
+TR::Register *OMR::ARM::TreeEvaluator::fmaxEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return NULL; 
+   }
+
+TR::Register *OMR::ARM::TreeEvaluator::fminEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return NULL; 
+   }
```

Comment:
These seem to be mapped already in the above define sequence:

```
#define fmaxARM64Evaluator TR::TreeEvaluator::unImpOpEvaluator 
#define dmaxARM64Evaluator TR::TreeEvaluator::unImpOpEvaluator 
```

Do we need these two functions then?

Reply:
I think [*compiler/arm/codegen/OMRTreeEvaluatorTable.hpp*](https://github.com/eclipse/omr/blob/f9ef03b68a474ae911b9dfac8e6e59c8c5c088b8/compiler/arm/codegen/OMRTreeEvaluatorTable.hpp#L710) contains:

```cpp
#define fmaxARMEvaluator TR::TreeEvaluator::fmaxEvaluator
#define dmaxARMEvaluator TR::TreeEvaluator::fmaxEvaluator
```

And we also have two (mutually exclusive) method definitions based on `#if (defined(__VFP_FP__) && !defined(__SOFTFP__))` on [Line 40](https://github.com/eclipse/omr/blob/f9ef03b68a474ae911b9dfac8e6e59c8c5c088b8/compiler/arm/codegen/FPTreeEvaluator.cpp#L40)

Reply:
Right! This is the ARM evaluators, and the defines were in ARM64. I made a mistake thinking they were in the same file. I rescind my review comment. All is well!